examples/airflow/README.md

```diff
@@ -1,43 +1,122 @@
-# Debugging [Airflow](https://airflow.apache.org) DAGs with [Marquez](https://marquezproject.github.io/marquez)
+# [Airflow](https://airflow.apache.org) Example
 
-In this example, you'll be introduced to the core concepts of Airflow: _DAGs_, _operators_, and _tasks_. You’ll write your very first DAG and learn how workflows are defined and executed in Airflow using pre-defined datasets and sources.
+In this example, we'll walk you through how to enable an **Airflow DAG** to send lineage metadata to **Marquez**.
 
-## Quickstart
+**What you’ll learn:**
 
-The easiest way to get up and running is with Docker. To start Airflow, run:
+* Collect DAG Metadata with Marquez
+* Explore inter-DAG dependencies using Marquez
+
+> **Note:** We've added [`marquez-airflow`](https://github.com/MarquezProject/marquez/tree/main/integrations/airflow) to `requirements.txt` to automatically send DAG metadata to Marquez.
+
+## Step 1: Start Airflow
+
+To start Airflow, run:
 
 ```bash
 $ ./docker/up.sh
 ```
 
 > **Tip:** Use the `--pull` to pull a tagged image.
```

Comment:
```suggestion
> **Tip:** The docker script also accepts parameters for  specific versions of airflow and other helpful commands. Information to use the script can be found [here](here).
```

Reply:
I removed the script from the example. One less hidden step.